helper-cli/src/main/kotlin/Main.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2019 HERE Europe B.V.
+ * Copyright (C) 2017-2019 HERE Europe B.V.
```

Comment:
Could you please do this throughout the code case? We have more matches in the analyzer and reporter-web-app modules.